authors/apps/authentication/backends.py

```diff
@@ -8,3 +8,6 @@
 
 """Configure JWT Here"""
 
+class JWTAuthentication:
```

Comment:
Are these changes part of the upgrade?

Reply:
Yes, they are. It's a way to deal with some errors we were getting.

Reply:
We needed to write JWTAuthentication class to remove errors that were preventing project from running.